client/src/app/gateways/repositories/motions/motion-repository.service/motion-repository.service.ts

```diff
@@ -246,7 +246,6 @@ export class MotionRepositoryService extends BaseAgendaItemAndListOfSpeakersCont
             reason: partialMotion.reason,
             number: partialMotion.number,
             state_extension: partialMotion.state_extension,
-            statute_paragraph_id: partialMotion.statute_paragraph_id,
```

Comment:
Please remove the whole function createStatuteAmendment(partialMotion: Partial<Motion & { workflow_id: Id }>)(see line 326)
This function is twice in the code, also amendment-controller.service.ts
and remove the state 'AmendmentAction.CREATE_STATUTEBASED_AMENDMENT'

Reply:
done